src/ol/Object.js

```diff
@@ -133,7 +133,14 @@ class BaseObject extends Observable {
    * @api
    */
   getProperties() {
-    return assign({}, this.values_);
+    return (this.values_ && assign({}, this.values_)) || {};
```

Comment:
I see that to maintain backwards compatibility here we should return `{}` instead of `undefined` (so never mind https://github.com/openlayers/openlayers/pull/11188/files#r444566590).